updater/reports/ReportReposPersonalNonOwnerPush.py

```diff
@@ -0,0 +1,37 @@
+from .ReportDaily import *
+
+# Find personal repositories that non-owners are pushing to.
+# These kind of repositories should be moved into organizations.
+# Only look at active users (not suspended!) and only look at pushes
+# of the last 4 weeks.
+class ReportReposPersonalNonOwnerPush(ReportDaily):
+	def name(self):
+		return "repositories-personal-non-owner-push"
+
+	def updateDailyData(self):
+		self.detailedHeader, self.detailedData = self.parseData(self.executeQuery(self.query()))
+		if len(self.data) == 0:
+			self.header = ["date", "personal repositories with non-owner pushes"]
+		self.data.append([str(self.yesterday()), len(self.detailedData)])
+		self.truncateData(self.timeRangeTotal())
+		self.sortDataByDate()
+
+	def query(self):
+		query = '''
+			SELECT
+				CONCAT(users.login, "/", repositories.name) as "repository"
+			FROM
+				repositories
+				JOIN users ON repositories.owner_id = users.id
+				JOIN pushes ON pushes.repository_id = repositories.id
+			WHERE
+				    users.type = "user"
+				AND users.suspended_at IS NULL
+				AND pushes.created_at > DATE_SUB(NOW(), INTERVAL 4 WEEK)
```

Comment:
As I suggested [in my comment on #95](https://github.com/Autodesk/hubble/pull/95#discussion_r164419517), I’d not use `NOW()` etc.